verification/curator-service/ui/src/components/new-case-form-fields/Symptoms.tsx

```diff
@@ -11,7 +11,7 @@ import { useFormikContext } from 'formik';
 
 // TODO: get values from DB
```

Comment:
nit but should we really be getting those 3 from the DB? I don't think so.

Reply:
If we are going to verify the field is one of these 3 values in the backend, it would be nice to only have the values defined in one location eventually. Agree it's not high priority.